SPECS/shadow-utils/shadow-utils.spec

```diff
@@ -22,6 +22,7 @@ Source12:       useradd-default
 Source13:       login-defs
 Patch0:         chkname-allowcase.patch
 Patch1:         libsubid-pam-link.patch
+Patch2:         CVE-2023-29383.patch
```

Comment:
I do not think this spec uses `%autosetup`, we might need to add an explicit `%patch2 -p1`

Reply:
Ah yes! Good catch. Will fix and rerun the builds.